src/test/CalculatorPage.test.js

```diff
@@ -0,0 +1,14 @@
+import { render, screen } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import CalculatorPage from '../pages/CalculatorPage';
+/* eslint-disable no-unused-expressions */
+
```

Comment:
- [ ] Please do not disable linters because it helps us to write better and clean codes. Kindly fix the linter error instead of disabling it.

Reply:
Resolved 👍🏽